services/file_get_info.proto

```diff
@@ -32,75 +42,109 @@ import "query_header.proto";
+     * This includes the signed payment and what kind of response is requested
+     * (cost, state proof, both, or neither).
      */
     QueryHeader header = 1;
 
     /**
-     * The file ID of the file for which information is requested
+     * A file identifier.
+     * <p>
+     * This MUST be the identifier of a file that exists in HFS.<br/>
+     * This value SHALL identify the file to be queried.
      */
     FileID fileID = 2;
 }
 
 /**
- * Response when the client sends the node FileGetInfoQuery
+ * A response to a query for the metadata of a file in the HFS.
  */
 message FileGetInfoResponse {
     /**
-     * Standard response from node to client, including the requested fields: cost, or state proof,
-     * or both, or neither
+     * The standard response information for queries.<br/>
+     * This includes the values requested in the `QueryHeader`
+     * (cost, state proof, both, or neither).
      */
     ResponseHeader header = 1;
 
-
     message FileInfo {
         /**
-         * The file ID of the file for which information is requested
+         * A file identifier.
+         * <p>
+         * This SHALL be the identifier of a file that exists in HFS.<br/>
+         * This value SHALL identify the file that was queried.
          */
         FileID fileID = 1;
 
         /**
-         * Number of bytes in contents
+         * A size, in bytes, for the file.
          */
         int64 size = 2;
 
         /**
-         * The current time at which this account is set to expire
+         * An expiration timestamp.
+         * <p>
+         * The file SHALL NOT expire before the network consensus time
+         * exceeds this value.<br/>
+         * The file SHALL expire after the network consensus time
+         * exceeds this value.<br/>
          */
         Timestamp expirationTime = 3;
 
         /**
-         * True if deleted but not yet expired
+         * A flag indicating this file is deleted.
+         * <p>
+         * A deleted file SHALL have a size `0` and empty content.
          */
         bool deleted = 4;
 
         /**
-         * One of these keys must sign in order to modify or delete the file
+         * A KeyList listing all keys that "own" the file.
+         * <p>
+         * All keys in this list MUST sign a transaction to append to the
+         * file content, or to modify file metadata.<br/>
+         * At least _one_ key in this list MUST sign a transaction to delete
+         * this file.<br/>
+         * If this is an empty `KeyList`, the file is immutable, cannot be
+         * modified or deleted, but MAY expire. A `fileUpdate` transaction MAY
+         * extend the expiration time for an immutable file.
          */
         KeyList keys = 5;
 
         /**
-         * The memo associated with the file
+         * A short description for this file.
+         * <p>
+         * This value, if set, SHALL NOT exceed 100 bytes when encoded as UTF-8.
```

Comment:
The size limit is configurable but defaults to 100 bytes. For parameters such as this that are configurable but unlikely to change, do we want to call them out as configurable?

Reply:
In most cases we should.  This is in a lot of places, however, so it might be better to make that change if/when we change the value.
I suppose which is better depends on how likely it is we'll ever change the value of that configuration.

Reply:
Will fix this wherever found (it's in a lot of places).